internal/scripts/deploy-dotcom.ts

```diff
@@ -62,7 +62,7 @@ const env = makeEnv([
 	'VERCEL_TOKEN',
 	'VITE_CLERK_PUBLISHABLE_KEY',
 	'WORKER_SENTRY_DSN',
-	previewId ? 'BOTCOM_POSTGRES_CONNECTION_STRING_PREVIEW' : 'BOTCOM_POSTGRES_CONNECTION_STRING',
+	previewId ? 'NEON_PREVIEW_DB_CONNECTION_STRING' : 'BOTCOM_POSTGRES_CONNECTION_STRING',
```

Comment:
This makes me a bit uneasy, don't really like that we might fall through here if we somehow fail to get `previewId`. We would fall back to the staging's connection string, so I guess it's fine.